srcpkgs/conky-cli/template

```diff
@@ -35,4 +38,5 @@ post_install() {
 	mv ${DESTDIR}/usr/share/doc/conky-cli/conky_no_x11.conf ${DESTDIR}/etc/conky/
 	rm ${DESTDIR}/usr/share/doc/conky-cli/convert.lua
 	vlicense COPYING
+	vman ../conky.1
```

Comment:
we should use create_wrksrc=yes to make sure everything is in one directory that gets cleaned after the build

Reply:
The part that fails, the gradient test respectively gradienth have been added just 23 days ago, so that is a very recent addition. It is not present in 1140 that version should build with no patch for 32 bit systems too (I think, I didn't build it yet)